src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1554901333.inc.php

```diff
@@ -0,0 +1,10 @@
+<h1>Build #1551195244</h1>
+<h2>Date: 2019-02-26</h2>
+<div class="changelog">
+</div>
+<?php
+
+TCMSLogChange::addInfoMessage('Pages are now only available if they are part of a navigation (see navigation
```

Comment:
How big is the probability (risk) that this is actually the case?

Reply:
I'm afraid I have no idea. Some developers might not be aware that this was a BC break, so I think there could be some systems with pages that will no longer be available. But that's why update messages are added in cases we think are important enough to justify behavior changes even in fix releases.